app/src/main/java/com/devshish/internship/presentation/ui/lyrics/LyricsFragment.kt

```diff
@@ -49,6 +53,14 @@ class LyricsFragment : Fragment(R.layout.fragment_lyrics) {
                 }
             }
 
+            isLyricsDeleted.observe(viewLifecycleOwner) {
+                it.getContentIfNotHandled()?.let {
+                    Snackbar.make(requireView(), R.string.lyrics_deleted, Snackbar.LENGTH_LONG)
+                        .setAnchorView(R.id.bottomNavView)
+                        .show()
+                }
+            }
+
```

Comment:
You have almost same code several lines highter - how about change this to work with only one event